### PR TITLE
Add ctrl-h to show/hide all panels

### PR DESCRIPTION
**Pull request by PhilPJL**

### CODE REVIEW

**Comment by ThyWoof** [comment]
@PhilPJL : shouldn't we add the same patch to GameLocationScreenBattle? Copy and paste from GameLocationScreenExploration renaming the class.

File: SolastaCommunityExpansion/Patches/GameUiLocation/GameLocationScreenExplorationPatcher.cs [hunk 1]
```diff
@@ -23,56 +23,88 @@ internal static class GameLocationScreenExploration_HandleInput
...
-                    else
+                    ShowPartyControlPanel();
+                    break;
+
+                case GameUiContext.CTRL_H:
+                    ShowAll();
+                    break;
+            }
+
+            void ShowAll()
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Patches/GameUiLocation/GameLocationScreenExplorationPatcher.cs:46]: The way this is currently implemented it looks like you could get into a weird state of showing half and hiding half of the panels by hitting ctrl+h. Should it be able to specify all hide/all show?

Comment (PhilPJL) [reply SolastaCommunityExpansion/Patches/GameUiLocation/GameLocationScreenExplorationPatcher.cs:46]: Good catch.

**Comment by PhilPJL** [comment]
> @PhilPJL : shouldn't we add the same patch to GameLocationScreenBattle? Copy and paste from GameLocationScreenExploration renaming the class.

Done.

> In that case I would move the entire logic to the Context and simply call the same method from both.

Not done yet.

**Review by ThyWoof — APPROVED** [review]

**Pull request merged**